photutils/psf/photometry.py

```diff
@@ -355,8 +355,8 @@ def nstar(self, image, star_groups):
             try:
                 from astropy.nddata.utils import NoOverlapError
             except ImportError:
-                raise ImportError("astropy 1.2.1 is required in order to use this
-                                   class.")
+                raise ImportError("astropy 1.2.1 is required in order to use" +
```

Comment:
super nitpicking: if a string is within parenthesis, you don't need the ``+``, just end the line with the quote mark, and start the new line with it (just as you did).